scripts/code_generation/jsoncan/src/can_database.py

```diff
@@ -253,6 +254,20 @@ def node_alerts(self, node: str, alert_type: CanAlertType) -> List[str]:
             if node in self.alerts
             else []
         )
+        
+    def node_id_codes(self, node: str, alert_type :CanAlert) -> Dict[str,int]:
+        
```

Comment:
Can you add a short function description like the other functions in this file?

Reply:
Yeah just added that. Sorry about that